app/models/accion.rb

```diff
@@ -2,13 +2,15 @@ class Accion < ActiveRecord::Base
   has_many :accion_en_snapshots
 
   def self.buscar_o_crear_desde_dato(dato)
-    find_by(id: dato['id']) || crear_desde_dato(dato)
+    id = dato['accion_id'] || dato['id']
+    nombre = dato['name'] || dato['nombre']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.